spec.bs

```diff
@@ -1045,7 +1139,63 @@ Note: Previously, the `payment` extension allowed for the creation of
               enrollment.https.html
             </wpt>
 
-:  <dfn export>Client extension processing ([=authentication extension|authentication=])</dfn>
+    1. Before step 13 (creating a {{CollectedClientData}}).
+
+        1. Let |bbk_allowed_algorithms| be {{AuthenticationExtensionsPaymentInputs/browserBoundPubKeyCredParams}}.
+
+            1. When
+                {{AuthenticationExtensionsPaymentInputs/browserBoundPubKeyCredParams}}
+                is empty, let |bbk_allowed_algorithms|
+                be
+                {{PublicKeyCredentialCreationOptions}}/{{PublicKeyCredentialCreationOptions/pubKeyCredParams}}.
+
+        1. Let (|bbk| - the platform specific key pair handle, |bbk_id| - an
+            identifier of the |bbk|) be the result of [=creating a key pair=]
+            using |bbk_allowed_algorithms|.
+
+            1. If [=create a key pair=] failed, skip steps below relating to
+                |bbk| and |bbk_public_key|.
+
+        1. [=Get a browser bound public key=] and let |bbk_public_key| be the result.
+
+    1. In step 13, instead of creating the {{CollectedClientData}} create a
+        {{CollectedClientPaymentData}} whose fields are:
+
+        1. {{CollectedClientPaymentData/payment}} set to a
+            {{CollectedClientAdditionalPaymentRegistrationData}} with
+
+            : {{CollectedClientAdditionalPaymentRegistrationData/browserBoundPublicKey}}
+            :: |bbk_public_key| - The COSE_Key encoded public key.
+
+        1. All other fields set as per the original step 13.
+
+    1. In [=create credential async loop|step 22=], case "any authenticator
+        indicates success", step 3, just before returning <var
+        ignore>pubKeyCred</var>:
+
+        1. [=Bind a key pair=] using |bbk_id| and <var
+            ignore>pubKeyCred</var>.{{PublicKeyCredential/[[identifier]]}}.
```

Comment:
@domfarolino - I'm not super used to internal slots, but looking at WebAuthn's spec, should this be worded as "and the <var ignore>pubKeyCred's</var> {{PublicKeyCredential/[[identifier]]}} internal slot." ?

(I.e., is it allowed to 'reference' internal slots via the `.` notation, or does one have to be more wordy as I've described above).